POLICIES.rdoc

```diff
@@ -0,0 +1,61 @@
+== Long-Term Support
+
+RubyGems long-term support policy will be similar to and follow the Ruby
+long-term support policy.
+
+When a Ruby version reaches end-of-life, the RubyGems version included in that
+release will also reach end-of-life.  After that date no security releases will
+be made for RubyGems releases up to the version shipped in the next release of
+Ruby.
+
+For example, Ruby 1.9.3 reached end-of-life on 23 February 2015.  RubyGems
+version 2.0.0 shipped with Ruby 2.0.0, so no RubyGems versions earlier than
+2.0.0 will receive any updates.  Ruby 2.1.0 shipped with RubyGems 2.2.0.  When
+Ruby 2.0.0 reaches end-of-life no RubyGems versions earlier than 2.2.0 will
+receive any updates.
+
+=== Security Fixes
+
+Security releases will be made for RubyGems version series that were included
+in a Ruby release.
+
+For example, RubyGems 2.0.x will recieve security fixes until Ruby 2.0.0
+reaches end-of-life.
+
+== Release Policies
+
+RubyGems primarily makes bug fix releases off of the master branch.  We may mix
+bug fixes and new features in the same release.  RubyGems does not guarantee
+it will make bug fix releases for earlier versions.
```

Comment:
Are new features and possibly non-backwards compatible changes made on master as well?